Shipr-Helm/templates/redis-service.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{  .Values.redis.name }}
+spec:
+  selector:
+    app: {{  .Values.redis.name }}
```

Comment:
can users not use others labels to select?

Reply:
Applied the condition for this